tests/test_tpp_backend.py

```diff
@@ -5858,3 +5859,152 @@ def get_cursor(self):
         pass
 
     assert attempts == 2
+
+
+def test_ons_cis():
+    session = make_session()
+    session.add_all(
+        [
+            Patient(
+                ONS_CIS=[
+                    ONS_CIS(
+                        age_at_visit="20",
```

Comment:
Is this a string for testing some aspect of the method? We use `int`s elsewhere.

```suggestion
                        age_at_visit=20,
```

Reply:
No, it's an int in the raw table, will update it